backend/api/views/question_view.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.decorators import api_view
```

Comment:
I would rename this file. Suffix `_view` is redundant. I propose `questions.py`.

Same for `quizzes.py`